esmvalcore/_recipe.py

```diff
@@ -442,9 +442,9 @@ def _update_fx_files(step_name, settings, variable, config_user, fx_vars):
             settings['add_fx_variables']['fx_variables'].update({
                 fx_var: fx_info
             })
-
-    logger.info('Using fx_files: %s for variable %s during step %s',
-                pformat(settings['add_fx_variables']['fx_variables']),
+            logger.debug(
+                'Using fx_files: %s for variable %s during step %s',
+                pformat(fx_files),
```

Comment:
```suggestion
            logger.info(
                'Using fx files for variable %s during step %s: %s',
                variable['short_name'], step_name, pformat(fx_files))
```

I think it's good to write this to `info` after all. Apart from that I'm happy with the changes! :+1: